Libraries/LibWeb/CSS/CSSStyleProperties.cpp

```diff
@@ -553,6 +553,13 @@ Optional<StyleProperty> CSSStyleProperties::get_property_internal(PropertyID pro
             auto left = get_property_internal(PropertyID::PaddingLeft);
             return style_property_for_sided_shorthand(property_id, top, right, bottom, left);
         }
+        case PropertyID::OverflowClipMargin: {
+            auto top = get_property_internal(PropertyID::OverflowClipMarginTop);
+            auto right = get_property_internal(PropertyID::OverflowClipMarginRight);
+            auto bottom = get_property_internal(PropertyID::OverflowClipMarginBottom);
+            auto left = get_property_internal(PropertyID::OverflowClipMarginLeft);
+            return style_property_for_sided_shorthand(property_id, top, right, bottom, left);
```

Comment:
I missed it before but `overflow-clip-margin` only takes one length, right? I think you'd need to reject this if the sides aren't the same. I don't remember how that works.